libraries/extensions/ros2-bridge/msg-gen/src/types/primitives.rs

```diff
@@ -241,7 +241,7 @@ impl GenericString {
         // TODO: Assertion
         let value = Literal::string(value);
         if self.is_wide() {
-            quote! { ::widestring::U16String::from_str(#value) }
+            quote! { ffi::U16String::from_str(#value) }
```

Comment:
Could you clarify why this widestring change is required?

Reply:
So it seems that with the current `AMENT_PREFIX_PATH` we're parsing more messages in the CI/CD than before. Some of them containing U16String. 

It seemed that the previous implementation were dependent on a miss imported widestring package that cannot be found. 

I'm not exactly sure if there is another implementation to tokenize a U16 string into `chars: Vec>u16>` but this implementation keep the spirit of the previous implementation. But, ultimately, it seems that U16String is a bit experimental, when I search for the procedural macro calls.